NEWS.md

```diff
@@ -1 +1,3 @@
-* Incremental Release Notes here.
+## 2020-01-15 v0.0.1-SNAPSHOT
```

Comment:
Let's have only release notes here. Which will be updated during the release

Reply:
done